### ISSUE TITLE
same occuring event, different behavior?

### ISSUE THREAD

**Issue by panacist** [TeamNewPipe/NewPipe#4160]
### Version

- 0.19.8

### Steps to reproduce the bug

1. turn off internet connectivity
2. go to services other than youtube and mediaccc
3. go to trending page for example!
4. and there you have it!

### Expected behavior

should see a network error and a retry button to press
### Actual behaviour

get an error to report !

**Comment by opusforlife2** [comment]
Yeah this seems to be a bad design decision that's seen throughout the app. Turn off internet, and suddenly searching becomes an issue (this is fixed in a PR), scrolling through lists becomes an issue, and now we can see that kiosks in certain services also become an issue.

**Comment by ShareASmile** [comment]
Not a issue anymore for latest v0.26.1
but still a issue for preunified version based on 0.19.8 if it finds hopefully [a new maintainer](https://github.com/XiangRongLin/NewPipe-preuinified/issues/26) 
![Screenshot_20240127_234627_NewPipe v0_26_0](https://github.com/TeamNewPipe/NewPipe/assets/60492161/c1d3b581-33da-4888-a80d-e9859920eb05)
This is how trending page looks of peertube kiosk when internet is off
